### PR TITLE
Add api to get bundle CSV data from koji/brew

### PR DESCRIPTION
**Pull request by qixiang**
Add a new API in kojiservice to get CSV data of bundle image from
koji/brew service.

JIRA: CWFHEALTH-913

### CODE REVIEW

File: freshmaker/kojiservice.py [hunk 1]
```diff
@@ -367,6 +370,42 @@ def get_ocp_versions_range(self, build_nvr):
...
         return ocp_versions_range
 
+    @freshmaker.utils.retry(wait_on=(requests.Timeout, requests.ConnectionError), logger=log)
+    def get_bundle_csv(self, build_nvr):
+        """
+        Return CSV(clusterserviceversion) data of operator bundle build
+
+        :param str build_nvr: NVR of operator bundle build.
+        :return: CSV data or None when build is not bundle nor CSV data doesn't exist
+        :rtype: dict
```
Comment (apaplaus) [inline freshmaker/kojiservice.py:380]:
```suggestion
        :rtype: dict or None
```

**Review by apaplaus — APPROVED** [review]

File: freshmaker/kojiservice.py (hunk 1 above)
Comment (qixiang) [reply freshmaker/kojiservice.py]: Updated

**Comment by apaplaus** [comment]
LGFM

**Comment by gnaponie** [comment]
+1

**Review by gnaponie — APPROVED** [review]

**Pull request merged**